src/event/event-type.js

```diff
@@ -251,7 +251,11 @@ const CustomEventType: PKEventTypes = {
   /**
    * Fired when the drm license is responded from the DRM server
    */
-  DRM_LICENSE_LOADED: 'drmlicenseloaded'
+  DRM_LICENSE_LOADED: 'drmlicenseloaded',
+  /**
+   * Fired when the player changes its visibility state - either by tabbing in / out or scrolling in /out of view
+   */
+  VISIBILITY_CHANGE: 'visibilitychange'
```

Comment:
why do we need this here? we can pull it to kaltura-player and merge it in the events getter

Reply:
done